src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -118,44 +118,52 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AdjoinedNotFlagError" xml:space="preserve">
-    <value>Adjoined alias is not a flag</value>
+    <value>Adjoined alias is not a flag: '{0}'</value>
+    <comment>Error message displayed when the user provides an adjoined alias that is not a flag argument. {0} is a placeholder replaced by the user input argument (e.g. '-ab').</comment>
   </data>
   <data name="AdjoinedNotFoundError" xml:space="preserve">
-    <value>Adjoined flag alias not found</value>
+    <value>Adjoined flag alias not found: '{0}'</value>
+    <comment>Error message displayed when the user provides an adjoined flag alias argument that was not found. {0} is a placeholder replaced by the user input argument (e.g. '-ab').</comment>
   </data>
   <data name="AvailableArguments" xml:space="preserve">
     <value>The following arguments are available:</value>
+    <comment>Message displayed to inform the user about the available command line arguments.</comment>
   </data>
   <data name="AvailableCommandAliases" xml:space="preserve">
     <value>The following command aliases are available:</value>
+    <comment>Message displayed to inform the user about the available command line alias arguments.</comment>
   </data>
   <data name="AvailableCommands" xml:space="preserve">
     <value>The following commands are available:</value>
-    <comment>Commands the tool supports</comment>
+    <comment>Message displayed to inform the user about the available command line command arguments.</comment>
   </data>
   <data name="AvailableHeader" xml:space="preserve">
     <value>Available</value>
     <comment>As in "a new version is available to update to".</comment>
+    <comment>Label displayed when a new application package is available.</comment>

```

Comment:
Unlikely since Visual studio only allow one comment (and extra comments will be removed on save). But we can have multiple lines comments if we want to add separate information.